source/declarations.tex

```diff
@@ -3659,9 +3661,13 @@
 argument and are not function parameter packs.
 Where syntactically correct and where ``\tcode{...}'' is not
 part of an \grammarterm{abstract-declarator},
-``\tcode{, ...}''
+``\tcode{...}''
 is synonymous with
-``\tcode{...}''.
+``\tcode{, ...}''.
+A \grammarterm{parameter-declaration-clause}
+of the form
+\grammarterm{parameter-declaration-list} \tcode{...}
+is deprecated.
```

Comment:
Should we have a iref to [depr.ellipsis.comma]?

Reply:
sounds good

Reply:
Can we actually do that on this PR? I have been told that any editorial fixes that would change the approved wording should be done as a separate follow-up PR.

Reply:
Done.